styles/todo-styles.css

```
@@ -0,0 +1,65 @@
+ul {
+  list-style-type: none;
+}
+
+li {
+  padding: 4px 10px;
+  border-radius: 5px;
+}
+
+li:hover {
+  background-color: #eee;
+}
+
+button {
+  border-radius: 5px;
+  padding: 5px;
+  transition: 0.5s;
+}
+
+.btn-add{
+  background: white;
+  border: solid 1px green;
+  color:green;
+  margin-left: 10px;
+}
+
+.btn-add:hover{
+  background: green;
+  color: white;
```

You should be able to create a selector that will target all buttons in hover state. Then you don't have to add it to each hover class

```suggestion
  button:hover{
    color:white;
  }
```

---

Good call